robotframework_reportportal/result_visitor.py

```diff
@@ -113,14 +112,22 @@ def end_keyword(self, kw):
         listener.end_keyword(kw.name, attrs)
 
     def start_message(self, msg):
-        if (msg.message != ''):
+        if msg.message != '':
```

Comment:
```suggestion
        if msg.message:
```

Reply:
Seems it's not been resolved.

Reply:
You're right, I think I accidentally override it when rebasing.